src/main/java/org/jvnet/hudson/test/TestPluginManager.java

```diff
@@ -113,6 +129,19 @@ private TestPluginManager() throws IOException {
         return names;
     }
     
+    /**
+     * Install a plugin from the resources directory.
+     * @param pluginName The plugin name.
+     * @throws IOException Error copying plugin.
+     */
+    public void installResourcePlugin(String pluginName) throws Exception {
+        URL res = TestPluginManager.class.getClassLoader().getResource("plugins/" + pluginName);
+        if (res == null) {
+            Assert.fail("Plugin '" + pluginName + "' not found in /resources/plugins.");
```

Comment:
:ant: Why `/resources`? Potentially misleading.

BTW easier to use `assertNotNull`.